Screens/ImageCapture.js

```diff
@@ -0,0 +1,70 @@
+import React, { Component } from 'react'
+import {
+  StyleSheet,
+  Text,
+  View,
+} from 'react-native'
+import Camera from 'react-native-camera'
+
+class ImageCapture extends Component {
+  state = {
+    images: []
+  }
+
+  static navigationOptions = {
+    title: 'Camera'
+  }
+
+  takePicture() {
+    const options = {}
+    this.camera.capture({ metadata: options })
+      .then((data) => {
+        console.log('image captured')
+        console.log('data: ', data)
+        this.props.navigation.state.params.setImages(data)
+      })
+      .catch(err => console.error(err))
+  }
+
+  render() {
+    const { goBack } = this.props.navigation
+    return (
+      <View style={styles.container}>
+        <Camera
+          ref={(cam) => {
+            this.camera = cam
+          }}
+          style={styles.preview}
+          aspect={Camera.constants.Aspect.fill}>
+          <Text style={styles.capture} onPress={
+            this.takePicture.bind(this)}
+            >
+            Capture
+          </Text>
```

Comment:
Haven't run this app yet - would an icon be better here - e.g., something from react-native-vector-icons?

Reply:
Or better yet - borrow some assets from the `react-native-camera` example here [https://github.com/lwansbrough/react-native-camera/tree/master/Example/assets](https://github.com/lwansbrough/react-native-camera/tree/master/Example/assets)

---
In reply to: [157103391](https://github.com/rozele/React-Native-Ski-Team/pull/2#discussion_r157103391) [](ancestors = 157103391)